src/NuGet.Core/NuGet.Common/Telemetry/TelemetryActivity.cs

```diff
@@ -61,8 +61,8 @@ public void Dispose()
             if (NuGetTelemetryService != null && TelemetryEvent != null)
             {
                 var endTime = DateTimeOffset.Now;
-                TelemetryEvent["StartTime"] = _startTime.ToString();
-                TelemetryEvent["EndTime"] = endTime.ToString();
+                TelemetryEvent["StartTime"] = _startTime.UtcDateTime.ToString("O");
+                TelemetryEvent["EndTime"] = endTime.UtcDateTime.ToString("O");
```

Comment:
Change `DateTimeOffset.Now` to `DateTimeOffset.UtcNow` instead for both `endTime` and `_startTime` above. `Now` gets the UTC time and then adjusts to local, so we can avoid making timezone adjustments twice per variable.